src/docfx/build/manifest/DependencyMapBuilder.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal class DependencyMapBuilder
```

Comment:
>DependencyMapBuilder [](start = 19, length = 20)

nit: document that this and (DependencyMap) is per file, I initially thought it was global (for all files)

Reply:
OK


---
In reply to: [193959556](https://github.com/dotnet/docfx/pull/2859#discussion_r193959556) [](ancestors = 193959556)